### DISCUSSION TITLE
Virus scan

### DISCUSSION THREAD

**Discussion by Zeamox** [kel-z/HSR-Scanner#69] [General]
Hey, I think that might be unintentional but it looks like some virus made it's way into the scanner-
Windows automatically deletes it, and VirusTotal detects 2 of them
![image](https://github.com/kel-z/HSR-Scanner/assets/82825374/008642f9-a86a-4847-ad0b-fdee810b3e72)

**Comment by kel-z** [comment]
The scanner has the capability to take mouse and keyboard control during the scan, which means that it can accidentally get flagged by anti-virus pretty easily.

There should be a way to whitelist the HSR-Scanner.exe so it doesn't get deleted, and the file should be safe to run as long as it was downloaded directly from this GitHub page (unless you don't trust that I didn't put a virus in it).

**Reply by Zeamox** [reply]
Ah okay I see, sorry for making a post about it then 😅
Just used it and it works just fine, thank you !

**Reply by kel-z** [reply]
No problem! Always better to be safe than sorry after all.
